### ISSUE TITLE
σˆk is the normalized σk

### ISSUE THREAD

**Issue by huimlight** [microsoft/SoftTeacher#100]
hello,  compute σˆk  use w+h as the denominator in paper, but σˆk use w,h,w,h in code. May I ask if I misunderstood?

**Comment by MendelXu** [comment]
Yes, the σˆk is a vector at first and then is averaged over w,h.
